sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/autorest.md

```diff
@@ -17,6 +17,10 @@ modelerfour:
   flatten-payloads: false
 skip-serialization-format-xml: true
 
+no-property-type-replacement: 
+- ApiManagementServiceIdentity
```

Comment:
This is not correct. I explained this in other comment. We should seek for other solutions

Reply:
Fxied